harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -416,7 +398,15 @@ def _train_for_step(self, step_id: int, batches_per_step: int) -> workload.Respo
                         parameters=parameters, divisor=self.hvd_config.aggregation_frequency
                     )
 
-                self._clip_grads(parameters)
+                # TODO: Remove this check in v0.12.8.
```

Comment:
blocking: This looks like a breaking change since we no longer use these hparams. Should go through the breaking change process as in https://docs.google.com/document/d/1P4ym1eF3LosYe_hGAtFqXyt8PhyPqTmEihiPtG8tnm8/edit.

Reply:
Updated.